app/frontend/src/css/application.scss

```diff
@@ -13,6 +13,9 @@ $moj-assets-path: "~@ministryofjustice/frontend/moj/assets/";
 .nojs-hidden {
   display: none;
 }
+.moj-action-bar__filter:after {
```

Comment:
I believe this would affect all .moj-action-bar__filters. I added a rule on line 231 that may help on a more granular level.